microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -1050,6 +1054,51 @@ If your device is managed by using group policies set by your organization, the
+
+The following writing tasks are supported:
+
+<!-- todo: 'task' vs. 'action', in context of whitepaper rather than design doc
+The terminology usage could be either:
+* a task is accomplished by an action/gesture
+* an action is accomplished by a gesture
+-->
+
+| Task | Action | Before | After |
+|---|---|---|---|
+| Insert text | Write in or near the text box. | ![Insert text (before)](./index-images/Abydos-add-text-before.png) | ![Insert text (after)](./index-images/Abydos-add-text-after.png) |
+| Delete text | Scribble over the words that you want to delete. | ![Delete text (before)](./index-images/Abydos-delete-text-before.png) | ![Delete text (after)](./index-images/Abydos-delete-text-after.png)|
+| Add space | Draw a vertical line where you want a space to be inserted. | ![Add space (before)](./index-images/Abydos-insert-space-before.png) | ![Add space (after)](./index-images/Abydos-insert-space-after.png) |
+| Remove space | Draw a vertical line on the space that you want to remove. | ![Remove space (before)](./index-images/Abydos-remove-space-before.png) | ![Remove space (after)](./index-images/Abydos-remove-space-after.png) |
+| Select text | Draw a horizontal line across the text to select. | ![Select text (before)](./index-images/Abydos-select-text-before.png) | ![Select text (after)](./index-images/Abydos-select-text-after.png) |
+| Add newline | Draw a "backwards L" at the end of a line of text. | ![Add newline (before)](./index-images/Abydos-add-newline-before.png) | ![Add newline (after)](./index-images/Abydos-add-newline-after.png) |
+<!-- todo: rename pngs lowercase and not Abydos product code name per style guide: https://review.learn.microsoft.com/help/platform/seo-optimize-urls?branch=main#name-your-files -->
+
+Aggregate feature usage data is collected to measure frequency of usage and latency.  Text that you've inputted and personal identifiers are not sent to Microsoft.
+
+To opt out of Windows Ink for Microsoft Edge, go to `edge://settings/content/HandwritingToText`:
+<!-- todo: m.h.: first give the basic GUI/WIMP way, before the special URL way-->
+<!-- todo: edge://settings/content/HandwritingToText not found on m.h. machine (end up at edge://settings/profiles), is it in Stable yet? -->
+
+![Windows Ink settings in Microsoft Edge](./index-images/Abydos-edge-settings.png)
+<!-- todo: m.h.: re-create using standard style for screenshots -->
+
+To disable Windows Ink for an entire Windows device, go to **Settings > Bluetooth & devices > Pen & Windows Ink > Shell Handwriting**:
+<!-- todo: Shell Handwriting not found on m.h. machine, is it in Stable yet? -->
+
+![Windows Ink settings in Windows](./index-images/Abydos-windows-settings.png)
+<!-- todo: m.h.: re-create using standard style for screenshots -->
```

Comment:
I recommend removing these screenshots for parity with other features' "disable" directions in the public privacy whitepaper.

```suggestion
To disable Windows Ink for Microsoft Edge, go to `edge://settings/content/HandwritingToText` and turn off the toggle for **Turn handwriting into text on input fields**. To disable Windows Ink for an entire Windows device, go to **Settings > Bluetooth & devices > Pen & Windows Ink > Shell Handwriting** and turn off the toggle for **Use your handwriting to enter text**.
<!-- todo: m.h.: first give the basic GUI/WIMP way, before the special URL way-->
<!-- todo: edge://settings/content/HandwritingToText not found on m.h. machine (end up at edge://settings/profiles), is it in Stable yet? -->
<!-- todo: Shell Handwriting not found on m.h. machine, is it in Stable yet? -->
```

Reply:
if we elim those Settings pngs, no need for me to re-create them w/ latest styles for screenshots

Reply:
Re: not seeing settings pages - it's only avail on Win 11 touch devices and not 100% in Stable yet. Lmk if that should be flagged anywhere in the text.

Reply:
The .pngs look ok, and this feature is not on my machine, so I can't provide (nice-to-do) styled pngs.   Removed todo comments.

Added "11": 
Windows Ink allows you to write with a digital pen directly into an input field on Windows 11 touch devices.
@majenny

Reply:
removed these two pngs, after making the UI text complete.